tests/libs/estdlib/test_gen_server.erl

```diff
@@ -126,6 +126,17 @@ test_late_reply() ->
     ok = gen_server:call(Pid, {reply_after, 0, ok}),
     {message_queue_len, 0} = erlang:process_info(self(), message_queue_len),
     %%
+    %% test that timed out message is properly flushed
+    %%
+    timeout =
+        try gen_server:call(Pid, {reply_after, 300, ok}, 200) of
+            unexpected -> unexpected
+        catch
```

Comment:
Nit: Might want to run this through `erlfmt`

Reply:
I believe this matches `erlfmt` style which is now enforced.